src/Lykke.HttpClientGenerator/Lykke.HttpClientGenerator.csproj

```diff
@@ -16,11 +16,11 @@
   <ItemGroup>
     <PackageReference Include="Autofac" Version="4.8.1" />
     <PackageReference Include="JetBrains.Annotations" Version="11.1.0" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.0.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.1.1" />
     <PackageReference Include="Microsoft.Extensions.PlatformAbstractions" Version="1.1.0" />
-    <PackageReference Include="Polly" Version="5.8.0" />
-    <PackageReference Include="Polly.Caching.MemoryCache" Version="1.0.0" />
-    <PackageReference Include="Refit" Version="4.2.0" />
-    <PackageReference Include="System.Reflection.DispatchProxy" Version="4.4.0" />
+    <PackageReference Include="Polly" Version="6.0.1" />
+    <PackageReference Include="Polly.Caching.Memory" Version="2.0.0" />
+    <PackageReference Include="Refit" Version="4.5.6" />
+    <PackageReference Include="System.Reflection.DispatchProxy" Version="4.5.1" />
   </ItemGroup>
```

Comment:
Enable XML documentation in the project settings

Reply:
it was already on true